src/handlers/canonicalFieldsHandlers.ts

```diff
@@ -1,13 +1,35 @@
+import { logger } from '../logger';
+import { db } from '../database';
+import { isCanonicalFieldArray } from '../types';
+import { ValidationError } from '../errors';
 import type {
   Request,
   Response,
 } from 'express';
 
-function getCanonicalFields(req: Request, res: Response): void {
-  res.status(200)
-    .contentType('application/json')
-    .send('{ "hello": "world"}');
-}
+const getCanonicalFields = (req: Request, res: Response): void => {
+  db.sql('canonicalFields.fetchAll')
+    .then((canonicalFields) => {
+      logger.debug(canonicalFields, 'canonicalFields.fetchAll');
```

Comment:
@bickelj I'd love your thoughts on when it's good to have built in logger lines in code... This is something I've always been torn about.

Reply:
Whenever the developer working on that code finds it useful. I think if it was useful before, it is likely to be useful again. Until it's obviously in the way and clear enough, at which point we can remove it.

Reply:
My example in another ticket of logging each method entry is far too much to be pushed to main, though, that is something I would only do locally.